cleanlab_studio/studio/studio.py

```diff
@@ -396,6 +397,99 @@ def download_embeddings(
         """
         return np.asarray(api.download_array(self._api_key, cleanset_id, "embeddings"))
 
+    def create_enrichment_project(
+        self,
+        dataset_id: str,
+        name: str,
+        target_column_in_dataset: str = "metadata",
+    ) -> enrichment.EnrichmentProject:
+        """
+        Creates a Cleanlab Studio Enrichment Project.
+
+        Args:
+            dataset_id (str): ID of dataset to be enriched.
+            name (str): Name of the enrichment project to create.
+            target_column_in_dataset: (str): Name of the column to populate with new data. Defaults to "metadata".
+
+        Returns:
+            EnrichmentProject: The [EnrichmentProject](../enrichment#class-enrichmentproject) object
+            for the new enrichment project.
+        """
+        enrichment_project_dict = api.create_enrichment_project(
+            api_key=self._api_key,
+            dataset_id=dataset_id,
+            name=name,
+            target_column_in_dataset=target_column_in_dataset,
+        )
+
+        return enrichment.EnrichmentProject(
```

Comment:
this is what I mean. 
instead of manually put the dict value one by one. 
do `EnrichmentProject.from_dict(enrichment_project_dict)` hide everything into the object definition.

Reply:
Unfortunately `created_at_string` complicates this.

Reply:
I'm not sure I understand why created_at makes things complicate.
it's a value from backend with fixed format, `from_dict`or `from_json` can just adapt the format. no?

Reply:
I've updated `EnrichmentProject`s `created_at` arg to take string as well, and now do the conversion inside the class.